modular_skyrat/modules/pixel_shift/code/pixel_shift.dm

```diff
@@ -57,3 +70,19 @@
 			if(pixel_x >= -16 + base_pixel_x)
 				pixel_x--
 				is_shifted = TRUE
+
+	// Yes, I know this sets it to true for everything if more than one is matched.
+	// Movement doesn't check diagonals, and instead just checks EAST or WEST, depending on where you are for those.
+	if(pixel_y > 8)
+		passthroughable |= EAST | SOUTH | WEST
+	if(pixel_x > 8)
+		passthroughable |= NORTH | SOUTH | WEST
+	if(pixel_y < -8)
+		passthroughable |= NORTH | EAST | WEST
+	if(pixel_x < -8)
+		passthroughable |= NORTH | EAST | SOUTH
+
+/mob/living/CanAllowThrough(atom/movable/mover, border_dir)
+	if(passthroughable)
+		return passthroughable & border_dir
```

Comment:
I feel like this could be problematic when you're lying down, no? Like, the fact that it instantly returns sounds a bit odd to me, why wouldn't this work?
```suggestion
	if(passthroughable & border_dir)
		return TRUE 
```